my-app/src/display.js

```diff
@@ -15,33 +18,61 @@ function Display() {
 
   // Effect Hook: https://reactjs.org/docs/hooks-effect.html
   useEffect(() => {
-    geoFindMe();
+    var location =geoFindMe();
   });
 
+  const [searchParams] = useSearchParams();
+
   return (
-    <div >
-      WhatToEat
-      
-      <br />
+    <div style={{ background: '#EBE2D9' }} id="main">
+    <br />
+    <div class="WhatToEat"> 
+      What To
       <br />
+      Eat
+      </div>
       <p id="status"></p>
       <a id="map-link" target="_blank"></a>
-      <br />
-      <br />
-
-        <button onClick={() => navigateToCuisine('Chinese')}>
-            Chinese
-        </button>
-        <button onClick={() => navigateToCuisine('American')}>
-            American
-        </button>
-        <button onClick={() => navigateToCuisine('Mexican')}>
-            Mexican
-        </button>
-        <button onClick={() => navigateToCuisine('Korean')}>
-            Korean
-        </button>
-         
+      <div class="row">

```

Comment:
Is the same code duplicated in line 56 to 75?

Reply:
Yeah I'm just using it as placeholders for other cuisines so that I can format the CSS